imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -139,28 +139,28 @@ def calculate_calibration_factor(time):
     3. Linear interpolate between those two nearest time and get factor for input time.
 
     What this function is doing:
```

Comment:
Might need an extra linebreak here so the 1. isn't wrapped into this statement.

Also, the list above this might need to have the pipes too?
https://imap-processing--283.org.readthedocs.build/en/283/reference/generated/imap_processing.swe.l1b.swe_l1b_science.calculate_calibration_factor.html#imap_processing.swe.l1b.swe_l1b_science.calculate_calibration_factor

Reply:
fixed that